src/components/Skeletons/TableListItemSkeleton.tsx

```diff
@@ -1,17 +1,103 @@
 import React from 'react';
-import {Rect} from 'react-native-svg';
+import {Circle, Rect} from 'react-native-svg';
+import useThemeStyles from '@hooks/useThemeStyles';
+import useWindowDimensions from '@hooks/useWindowDimensions';
+import CONST from '@src/CONST';
 import ItemListSkeletonView from './ItemListSkeletonView';
 
 type TableListItemSkeletonProps = {
     shouldAnimate?: boolean;
     fixedNumItems?: number;
 };
 
+const circleRadius = 8;
+const padding = 12;
 const barHeight = '10';
 const shortBarWidth = '40';
 const longBarWidth = '120';
 
 function TableListItemSkeleton({shouldAnimate = true, fixedNumItems}: TableListItemSkeletonProps) {
+    const styles = useThemeStyles();
+    const {windowWidth, isSmallScreenWidth} = useWindowDimensions();
+    if (isSmallScreenWidth) {
+        return (
+            <ItemListSkeletonView
+                itemViewHeight={CONST.SEARCH_SKELETON_VIEW_ITEM_HEIGHT}
+                itemViewStyle={[styles.highlightBG, styles.mv2, styles.br3, styles.mr3, styles.ml3]}
+                shouldAnimate={shouldAnimate}
+                fixedNumItems={fixedNumItems}
+                renderSkeletonItem={() => (
+                    <>
+                        <Circle
+                            cx={padding + circleRadius}
+                            cy={padding + circleRadius}
+                            r={circleRadius}
+                        />
+
+                        <Rect
+                            x={padding + circleRadius * 2 + 4}
+                            y={padding + circleRadius - 2}
+                            width={windowWidth * 0.2}
```

Comment:
Could we just give this a fixed width of 40?

Reply:
I updated. Here is result:

![image](https://github.com/Expensify/App/assets/170341821/5f27c687-e42c-4cb3-b01c-c7792b166d17)

Reply:
This doesn't match the Figma design though. Can you double check? The receipt icon is 36x40.